images/kindnetd/cmd/kindnetd/masq.go

```diff
@@ -58,7 +60,9 @@ type IPMasqAgent struct {
 func (ma *IPMasqAgent) SyncRulesForever(interval time.Duration) error {
 	errs := 0
 	for {
-		if err := ma.SyncRules(); err != nil {
+		// get subnets each time (so we eventually reconcile)
+		noMasqueradeCIDRs := getNoMasqueradeSubnets(ma.clientset)
```

Comment:
we should check that this variable is not empty.
should we try to sync rules if is empty?

Reply:
For the reconcile pattern: yes. If there are no desired subnets to not masquerade, we should sync the rules.

Reply:
Imagine for testing I want to disable this. Right now I could eg remove the kubeadm configmap.